src/main/java/au/org/emii/wps/NetcdfOutputProcess.java

```diff
@@ -61,6 +61,8 @@ public NetcdfOutputProcess(WPSResourceManager resourceManager, Catalog catalog,
         this.catalog = catalog;
         this.context = context;
         this.workingDir = getWorkingDir(resourceManager);
+
+        logger.info("constructor");
```

Comment:
Not strictly necessary. Knowing the netcdf process was successfully constructed and when execute() is called will ease tracing (any) production issues.